encrypted-media-respec.html

```diff
@@ -1598,6 +1591,10 @@
         The above rule implies that the CDM instance must not be destroyed until all <a>MediaKeys</a>
         objects and all <a>MediaKeySession</a> objects associated with the CDM instance are destroyed.
       </p>
+      <p>
+        When a <a>MediaKeySession</a> object is destroyed, the CDM SHALL close the <a href="#key-session">key session</a> associated with the object.
```

Comment:
I think during garbage collection is how this will be implemented. Is there a common concept of doing something when the "object becomes inaccessible?" If so, is that actually implemented in browsers? It seems that would require integration with the reference counting and garbage collection logic.

At least for the current steps, do we need to be more specific about the timing? In cases that applications actually care, they should call `close()` and wait for the returned promise. See the last part of https://github.com/w3c/encrypted-media/issues/358#issuecomment-259223670.